src/domains/collections.ts

```diff
@@ -0,0 +1,256 @@
+import { gql, request } from "graphql-request";
+import {
+  Extents,
+  AssetLinks,
+  SpatialExtent,
+  STACCollection,
+} from "../@types/StacCollection";
+import {
+  Collection,
+  CollectionsInput,
+  FacetGroup,
+} from "../models/GraphQLModels";
+import { mergeMaybe } from "../utils";
+
+const CMR_ROOT = process.env.CMR_URL;
+
+import {
+  WHOLE_WORLD_BBOX,
+  pointStringToPoints,
+  parseOrdinateString,
+  addPointsToBbox,
+  mergeBoxes,
+  reorderBoxValues,
+} from "./bounding-box";
+
+const GRAPHQL_URL = process.env.GRAPHQL_URL ?? "http://localhost:3002/api";
```

Comment:
Is 3002 the default port for a local instance of graphql? In the serverless.yaml it is httpPort: 3013. https://github.com/nasa/cmr-graphql/blob/main/serverless.yml